demo

```diff
@@ -126,7 +126,7 @@ download_demo_artifact() {
 platform_dependent_setup() {
     if [[ "$OSTYPE" == "darwin"* ]]; then
         ARTIFACT_SIZE_BYTES=$(stat -f %z ${DEMO_ARTIFACT_NAME}) # BSD is not GNU -_-
-        export GATEWAY_IP=$(ifconfig | sed -En 's/127.0.0.1//;s/.*inet (addr:)?(([0-9]*\.){3}[0-9]*).*/\2/p' | head -1)
+        export GATEWAY_IP=$(ifconfig `netstat -rn | grep default | head -1 | awk '{print($NF)}'` inet | grep -F 'inet '| sed -e 's/.*inet \([^ ]*\) .*/\1/')
```

Comment:
Syntax:
Seems `$()` is used, and preferred for subcommands :)

Reply:
should now be `$()` 👌